aws/cloudformation/moz-single.json

```diff
@@ -484,6 +484,14 @@
             "Type": "A",
             "TTL": "30",
             "ResourceRecords" : [ { "Fn::GetAtt" : [ "FxaEc2Instance", "PublicIp" ] } ]
+          },
+          {
+            "Name" : { "Fn::Join" : [ "", ["static", ".", {"Ref" : "Subdomain"}, ".", {"Ref" : "HostedZone"}, "."]]},
+            "Type": "A",
+            "AliasTarget" : {
+              "HostedZoneId" : "Z2FDTNDATAQYW2",
```

Comment:
This HostedZoneId was specified in the Route53 docs as the ID to use for Cloudfront.